UI/src/app/components/panel/panel.component.html

```diff
@@ -1,4 +1,5 @@
-<mat-expansion-panel class="expansion-panel" (click)="onClickMe()">
+<mat-expansion-panel 
```

Comment:
no need in additional class. You have the only one mat-expansion panel in each component. Style this tag